mimic/model/nova_objects.py

```diff
@@ -822,7 +822,7 @@ def request_delete(self, http_delete_request, server_id):
         server.update_status(u"DELETED")
         return b''
 
-    def request_action(self, http_action_request, server_id):
+    def request_action(self, http_action_request, server_id, absolutize_url):
```

Comment:
@gabecase - if you look at what `absolutize_url` actually *is*, you will notice that it is a function which converts a relative URL to an absolute URL, *not* an absolute URL, so its name is accurately describing what it's doing; as it is a function and not data, a verb, not a noun, is appropriate.